lib/linter.js

```diff
@@ -299,6 +304,10 @@ exports.invoke = async function (cwd, args, text, hash, callback) {
     clearRequireCache(cwd);
     cache = createCache(cwd, eslint_path_arg);
   }
+  if (!cache) {
+    callback(text);
```

Comment:
I don't understand the intention behind returning the `text` here. Can you elaborate?

Reply:
When reaching this state we essentially want to do nothing. We can achieve this by either gracefully exiting or returning the provided text. This MR opts for the latter, mainly due to your previous comment, but I'm fine with either. The first option is preferred from a performance standpoint and the second for a dramatic standpoint.

Reply:
Yes, doing nothing is fine. But why return the text and not `null`?

Reply:
😅

I'm returning `text` here due to, what I now think, is a wrong assumption. I thought that returning `null` to the callback would return an empty result for the formatting.

Would you prefer to call the callback with `null` here or just gracefully exiting instead?

Reply:
Well, if the first argument to the callback is truthy, it's treated as an error and `eslint_d` will exit with a non-zero exit code. The second argument is text to be printed, but not to cause an error, like a linter warnings.

The `text` is a special feature and it contains text that was piped to `eslint_d` (e.g. `echo "something" | eslint_d --stdin`). I guess when you tested this locally, `text` was empty and therefore it works as expected. You should call the callback with `null` if you want to exit without an error in all cases.

Reply:
Cheers for the detailed explanation. I'll change it to `null` as soon as I have time! 👍

Reply:
This is now changed to using `null` instead.